### ISSUE TITLE
[v2.0.0] trace.js starts complaining about lots of things

### ISSUE THREAD

**Issue by NordlingArt** [NativeScript/NativeScript#2052]
Ever since I upgraded to 2.0.0 `trace.js` has started spamming the console about things like:

`CONSOLE ERROR file:///app/tns_modules/trace/trace.js:160:30: Style: Error setting property: backgroundColor view: TextField(30)@file:///app/views/login/login.xml:16:7; value: 0 Error: Invalid value 0 for property backgroundColor`

It's repetitive too. Same error for almost every XML element. I don't even have any background set in the XML file, neither in the CSS file.

For instance, this is the row that the above error is complaining about:

`<TextField text="{{ user.password }}" hint="{{ language.password }}" secure="true" ReturnKeyType="done" />`

Also, I don't have the trace module imported to any view. This just came up since the TNS upgrade. What is going on? Tested on iOS only.

**Comment by tsonevn** [comment]
Hi @NordlingArt , 
Thank you reporting this issue.

I was unable to reproduce this problem. Could you give me more info about the CLI version and your java version and how to replicate this behavior. I would appreciate if you send me sample project.
What I can suggest you at the moment to try removing the platform `tns platform remove ios` and than to add it again `tns platfrom add ios`
Let me know whether this helps or if I can assist you further. 

Regards, 
@tsonevn

**Comment by NordlingArt** [comment]
Hi @tsonevn,

I noticed I had put background-color: 0; in app.css. That's what the error complained about. However, I am still getting an error for registering new font:

`CONSOLE ERROR file:///app/tns_modules/trace/trace.js:160:30: Error: Error occur while registering font: The operation couldn’t be completed. (com.apple.CoreText.CTFontManagerErrorDomain error 105 - Could not register the CGFont '<CGFont (0x7c8bc100): FontAwesome>')`

CLI version is as stated in the title, 2.0.0. Java should be the latest as I reinstalled everything about a week ago. But I am testing on iOS though, so Java shouldn't matter?

I just created a completely new project from scratch using `tns create`. `tns platform remove/add ios` is therefor not the issue.

Sample project can be reached at: https://github.com/NordlingArt/tns-trace-issue

**Comment by tsonevn** [comment]
Hi @NordlingArt,
Thank you for the reply.
I tested your sample project and found that if you remove `fontModule.ios.registraterFont("FontAwesome.ttf")` from `app.js`, the error message about the font registering will stop

`app.js`

```
var application = require("application");
var fontModule = require("ui/styling/font");

application.start({ moduleName: "main-page" });
```
Indeed setting the font-family name in `app.css` would be enough to start using "FontAwesome" in your project.  
`app.css`

```
.title {
  font-family: "FontAwesome";
  font-size: 48;
  horizontal-align: center;
  vertical-align: center;
}
```
I hope this is applicable for you.

**Comment by NordlingArt** [comment]
Hi @tsonevn,

Oh, so you are saying that we don't need to use that line of code anymore? It was documented before, not sure what it's saying now. I'll try removing it later today. Thanks for the help!

**Comment by devna13** [comment]
Hi @tsonevn 

I tried your suggestions but I still get the error;

CONSOLE ERROR file:///app/tns_modules/trace/trace.js:160:30: Error: Error occur while registering font: The operation couldn’t be completed. (com.apple.CoreText.CTFontManagerErrorDomain error 105 - Could not register the CGFont '<CGFont (0x78abb160): FontAwesome>')

I checked the code there is no  fontModule.ios.registraterFont("FontAwesome.ttf") anywhere in the code. 

in fonts folder:
app/fonts/fontawesome-webfont.ttf
app/fonts/FontAwesome.otf

in app.css:
.fa{
font-family:"FontAwesome"
}

Any ideas?
